### PR TITLE
hubert + yashim/chore: add deustch language

### PR DESCRIPTION
**Pull request by yashim-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,6 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npm run pre-commit-translate
```
Comment (michio-rework) [inline .husky/pre-commit:4]: By removing these lines we are disabling key translation key extraction and linter on pre-commit, I think we need this.

**Review by michio-rework — CHANGES_REQUESTED** [review]

File: .husky/pre-commit (hunk 1 above)
Comment (yashim-deriv) [reply .husky/pre-commit:4]: my bad this was a test

**Review by habib-deriv — APPROVED** [review]
LGTM

**Review by habib-deriv — APPROVED** [review]
LGTM

**Pull request merged**